homeassistant/components/switch_as_x/config_flow.py

```diff
@@ -32,6 +32,7 @@
                 vol.Required(CONF_ENTITY_ID): selector.EntitySelector(
                     selector.EntitySelectorConfig(domain=Platform.SWITCH),
                 ),
+                vol.Required(CONF_INVERT): selector.BooleanSelector(),
```

Comment:
I'd think this should be an optional item with a default.

Reply:
Fixed 👍